File: hlife/src/main/java/com/chad/hlife/util/LogUtil.java
package com.chad.hlife.util;

import android.util.Log;

public class LogUtil {

    private static final boolean isShowLog = true;

    public static void v(String tag, String msg) {
        if (!isShowLog) {
            return;
        }
        Log.v(tag, msg);
    }

    public static void d(String tag, String msg) {
        if (!isShowLog) {
            return;
        }
        Log.d(tag, msg);
    }

    public static void i(String tag, String msg) {
        if (!isShowLog) {
            return;
        }
        Log.i(tag, msg);
    }


    public static void w(String tag, String msg) {
        if (!isShowLog) {
            return;
        }
        Log.w(tag, msg);
    }

    public static void e(String tag, String msg) {
        if (!isShowLog) {
            return;
        }
        Log.e(tag, msg);
    }
}

File: player/src/main/java/com/chad/player/util/LogUtil.kt
package com.chad.player.util

import android.util.Log

class LogUtil {

    private val isShowLog = true

    fun v(tag: String, msg: String) {
        if (!isShowLog) {
            return
        }
        Log.v(tag, msg)
    }

    fun d(tag: String, msg: String) {
        if (!isShowLog) {
            return
        }
        Log.d(tag, msg)
    }

    fun i(tag: String, msg: String) {
        if (!isShowLog) {
            return
        }
        Log.i(tag, msg)
    }

    fun w(tag: String, msg: String) {
        if (!isShowLog) {
            return
        }
        Log.w(tag, msg)
    }

    fun e(tag: String, msg: String) {
        if (!isShowLog) {
            return
        }
        Log.e(tag, msg)
    }
}